src/kata4.humanCatDogYears.js

```diff
@@ -1,5 +1,25 @@
 const humanCatDogYears = (number) => {
 
+  let hcdy = [];
+    hcdy.push(number);
+
+    const cat = function (number) {
```

Comment:
Really good thought process here in breaking the functionality up into different functions. I would probably not be afraid to make these more concise though:

```js
const cat = number => (((number - 2) * 4) + 24)
```